src/main/ts/Part2Ex3Optional.ts

```diff
@@ -57,10 +64,13 @@ TODO: use Optional.from to implement the following DOM function
  */
 
 export const getNextSibling = (e: Element): Optional<ChildNode> => {
-  throw new Error("TODO");
+  return Optional.from(e ? e.nextSibling : null);
```

Comment:
Why do you have a truthy check here as based on the types e cannot be nullable? As such, this could just be this:
```suggestion
  return Optional.from(e.nextSibling);
```